Misc/NEWS.d/next/Core and Builtins/2023-06-11-09-14-30.gh-issue-105331.nlZvoW.rst

```diff
@@ -0,0 +1 @@
+fix asyncio.sleep(float('nan')) does not raise ValueErro problem
```

Comment:
Please try harder here. The text requires proper punctuation, capitalization, markup, and it should describe the change in behavior rather than just stating "fix the <blank> problem".